jaeger/charts/linkerd-jaeger/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: linkerd-jaeger
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.9.1-edge
+version: 30.9.2-edge
```

Comment:
We should restore this change. We tend to update charts as part of releases.

Reply:
Reverted back the change as well as on the README file